client/src/questions-n-answers/sub-components/mini-components/addAnswerTable.jsx

```diff
@@ -4,7 +4,8 @@ import propTypes from 'prop-types';
 const AddAnswer = (props) => {
 
   return (
-    <table className={`answerTable ${props.classname}`}>
+    <div >
```

Comment:
I ran into issues with my table styling because my table was wrapped in a div. If you get similar issues, try using the React.fragment syntax we just learned about to wrap your table instead of wrapping with a div. Can use `<>` and `</>` as shorthand